### ISSUE TITLE
Back to gallery buttons

### ISSUE THREAD

**Issue by SIRVER21** [SIRVER21/splendor-AK#12]
Need to change so that the back to gallery button is bigger and easier to acces (add one more on the bottom) on the edit page.

Also, there is currently no back button on new card page. There should be one.
